sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>Is there any kind of copyright rule this image might conflict with? 🏴‍☠️</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Great point. I will find an image in public domain</p>